dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -54,6 +56,8 @@ internal class SdkFeature(
 ) : FeatureScope {
 
     internal val initialized = AtomicBoolean(false)
+    internal val contextUpdateListeners: ConcurrentHashMap<FeatureContextUpdateListener, Any?> =
```

Comment:
not sure if I should use a ConcurrentHashMap here or just a Set and make a copy when notifying the listeners to make sure it is concurrent